source/support/data-protection/index.html.md.erb

```diff
@@ -0,0 +1,162 @@
+---
+title: Data Protection for GOV.UK Account users
+weight: 32
+last_reviewed_on: 2021-02-02
+review_in: 6 months
+---
+
+# Respond to data protection requests
+
+When we use and hold user data, users gain rights under the UK General Data Protection Regulation (UK GDPR).
+Users make requests to exercise those rights and we have responsibilities to respond.
+
+Below are some of the scenarios we will have to to understand and respond to along with information on who
```

Comment:
Try to avoid "above" and "below" as they don't help users who use assistive tech

Reply:
thanks, amended. I should go back and re-read the style guide!